deid/dicom/actions.py

```diff
@@ -156,6 +157,12 @@ def jitter_timestamp(dicom, field, value):
         value = int(value)
 
     original = dicom.get(field,None)
+    original_date = datetime.datetime.strptime(original, "%Y%m%d")
```

Comment:
This is going to trigger an error if the field is not in the dicom, or the format doesn't match the one you have provided. What is the rationale for making it more complicated than adding the number of days to the original?